### PR TITLE
Brings back the logging config from the cfpb_django era

### PR DESCRIPTION
**Pull request by rosskarchner**
Provides more visibility while troubleshooting problems in production.

## Changes

- Restores the ability to log to send errors to the apache error log

## Testing

- I'm... not sure, exactly. I can say that I've made this change live on the servers while troubleshooting, as documented in recent postmortems

## Review

- @cfpb/cfgov-backends 

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/cfgov/settings/production.py [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
         'mail_admins': {
             'level': 'ERROR',
-            'filters': ['require_debug_false'],
-            'class': 'django.utils.log.AdminEmailHandler',
-            'include_html': True,
+            'filters': [],
+            'class': 'django.utils.log.AdminEmailHandler'
+        },
+        'console': {
+            'level': 'DEBUG',
```
Comment (willbarton) [inline cfgov/cfgov/settings/production.py]: We're sure we want to log DEBUG to the console?

**Comment by willbarton** [comment]
bender-applause_medium

**Comment by willbarton** [comment]
👍

**Comment by richaagarwal** [comment]
I didn't test this but it looks sane 👍

**Comment by willbarton** [comment]
bender-applause_medium

**Comment by contolini** [comment]
:shipit:

**Pull request merged**